backend/api/Database/Context/FlotillaDbContext.cs

```diff
@@ -70,6 +70,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
         modelBuilder.Entity<MissionRun>().OwnsOne(m => m.Map).OwnsOne(t => t.TransformationMatrices);
         modelBuilder.Entity<MissionRun>().OwnsOne(m => m.Map).OwnsOne(b => b.Boundary);
+        modelBuilder.Entity<MissionRun>().HasOne(m => m.Area).WithMany();
```

Comment:
Do we need to define this in the FlotillaDbContext as well as the database models?

Reply:
I didn't do any changes on this, my guess is that Arnt Erik's editor formatted this line and that's why this has been marked as changed (this probably happened in other areas of the code) @aestene

Reply:
I suspect a rebase might be needed here. These are not changes I made with these files. We made some changes with this file when reverting your previous fix @oysand and I think this branch might not contain them and it looks fishy.

Try a rebase @mrica-equinor and we will see after.